pulsar-client-cpp/wireshark/pulsarDissector.cc

```diff
@@ -295,22 +367,20 @@ void link_to_response_frame(proto_tree* cmd_tree, tvbuff_t* tvb, int offset, int
 //////////
 
 /* This method dissects fully reassembled messages */
-static int dissect_pulsar_message(tvbuff_t *tvb, packet_info* pinfo, proto_tree* tree,
-                               void* data _U_) {
+static int dissect_pulsar_message(tvbuff_t *tvb, packet_info* pinfo, proto_tree* tree, void* data _U_) {
     col_set_str(pinfo->cinfo, COL_PROTOCOL, "Pulsar");
 
     conversation_t* conversation = find_or_create_conversation(pinfo);
-    ConnectionState* state = (ConnectionState*) conversation_get_proto_data(conversation,
-                                                                            proto_pulsar);
-    if (state == NULL) {
+    auto state = (ConnectionState*) conversation_get_proto_data(conversation,proto_pulsar);
+    if (state == nullptr) {
         state = new ConnectionState();
         conversation_add_proto_data(conversation, proto_pulsar, state);
     }
 
-    uint32_t offset = FRAME_SIZE_LEN;
-    int maxOffset = tvb_captured_length(tvb);
+    gint offset = FRAME_SIZE_LEN;
+    guint maxOffset = tvb_captured_length(tvb);
 
-    uint32_t cmdSize = (uint32_t) tvb_get_ntohl(tvb, offset);
+    guint32 cmdSize = tvb_get_ntohl(tvb, offset);
     offset += 4;
```

Comment:
The same as I've said before. The type casting between `quint32` and `uint32_t` is safe. There is no need to modify the original code. Because for the same reason, if someone else see the following code

```c++
   // offset is uint32_t, while cmdSize is guint32 after your change
   offset += cmdSize;
``` 

He might also want to change the code back or add the casting like

```c++
offset += static_cast<uint32_t>(cmdSize);
```

Therefore, stop making these unnecessary changes.

Reply:
Addressed.